internal/parse/parser.go

```diff
@@ -69,25 +69,24 @@ const (
 	typeId
 )
 
-// ParsedExpr is the AST representation of an SQL expression.
-// It has a representation of the original SQL statement in terms of queryParts
-// A SQL statement like this:
+// ParsedExpr is the AST representation of an SQL expression. The AST is made up
+// of QueryParts. For example, a SQL statement like this:
 //
-// Select p.* as &Person.* from person where p.name = $Boss.Name
+// Select p.* as &Person.* from person where p.name = $Boss.col_name
```

Comment:
Should this `s/Name/col_name` be in a separate PR?

Reply:
Its a comment that was missed in a previous PR that pertains to the QueryParts so here is probably the best place for it